sql/core/src/test/resources/sql-tests/results/cross-join.sql.out

```diff
@@ -128,6 +128,7 @@ two	2	two	2	one	1	two	2
 two	2	two	2	three	3	two	2
 two	2	two	2	two	2	two	2
 
+
```

Comment:
I don't saw you commit change to `cross-join.sql`, why we have change to `cross-join.sql.out`?

Reply:
ummm. That's so weird. I'm not sure about it but maybe this while line was be generated when I regenerated the golden file. I'll recover it. =P